### PR TITLE
Feature/test

### PR DESCRIPTION
**Pull request by vincent-samuelsson-karlskogafhs**

### CODE REVIEW

File: sokoban/src/App.tsx [hunk 1]
```diff
@@ -5,23 +5,34 @@ import { GameBoard } from "./Globals";
 import "./game.css";
 
 const colors: any = {
-  w: "blue",
+  w: "red",
   b: "brown",
-  p: "green",
+  g: "green",
```
Comment (vincent-samuelsson-karlskogafhs) [inline sokoban/src/App.tsx:10]: what is this ? What have you done.

**Comment by Dakawe** [comment]
AWESOME CODE!

File: sokoban/src/App.tsx (hunk 1 above)
Comment (Dakawe) [reply sokoban/src/App.tsx:10]: AWESOME CODE!

Comment (khansadia) [reply sokoban/src/App.tsx:10]: Waooow

**Pull request merged**